addons/mail/tests/test_mail_gateway.py

```diff
@@ -299,6 +299,150 @@
 """
 
 
+MAIL_EML_ATTACHMENT = """Subject: Re: test attac
+From: William <whe@odoo.com>
```

Comment:
I would make the test mail anonymous (no real name, mail, email or IP addresses).